src/advancedHMC_MCMC.jl

```diff
@@ -598,6 +608,12 @@ function ahmc_bayesian_pinn_ode(prob::DiffEqBase.ODEProblem, chain;
         samples, stats = sample(hamiltonian, Kernel, initial_θ, draw_samples,
             adaptor; progress = progress, verbose = verbose)
 
+        @info("Sampling Complete.")
+        @info("Current Physics Log-likelihood : ", physloglikelihood(ℓπ, samples[end]))
+        @info("Current Prior Log-likelihood : ", priorweights(ℓπ, samples[end]))
+        @info("Current MSE against dataset Log-likelihood : ",
+            L2LossData(ℓπ, samples[end]))
```

Comment:
Is this intended?

Reply:
yes

Reply:
Seems that should be returned information rather than just printed

Reply:
i could do that, but the info would be returned only after the sampling. The reason ive chosen to display it is so that at the beginning of the sampling the values of different loglikelihoods are visible and if proportion wrt to each other is off (weak priors will need us to have larger L2 loglikelihoods) we can simply exit sampling and adjust stds,etc.

Reply:
the final values of loglikelihoods shows which objective has been optimized more.